newIDE/app/config-overrides.js

```diff
@@ -0,0 +1,20 @@
+const { override, addWebpackModuleRule } = require('customize-cra');
```

Comment:
This is fairly lighweight as a package and not maintained, we could consider doing without it maybe?

Or just copy paste their implementation:

https://github.com/arackaf/customize-cra/blob/master/src/overrides.js
https://github.com/arackaf/customize-cra/blob/master/src/customizers/webpack.js#L327

so that we're just "plain react-app-rewired" instead of "react-app-rewired + customize-cra"

Reply:
I can indeed do without, good catch